warp-core/src/main/scala/com/workday/warp/junit/HasTestId.scala

```diff
@@ -14,9 +16,9 @@ import java.lang.reflect.Method
   */
 trait HasTestId {
 
-  def getTestClass: Option[Class[_]]
+  def getTestClass: Try[Class[_]]
```

Comment:
i thought it would be good to make these `Try` instead of `Option` so we can have a failure message in some future work. the junit methods we are currently calling return java `Optional`